backend/resource.go

```diff
@@ -36,7 +36,7 @@ type CallResourceRequest struct {
 
 // SetHTTPHeader sets the header entries associated with key to the
 // single element value. It replaces any existing values
-// associated with key. The key is case insensitive; it is
+// associated with key. The key is case-insensitive; it is
 // canonicalized by textproto.CanonicalMIMEHeaderKey.
 func (req *CallResourceRequest) SetHTTPHeader(key, value string) {
```

Comment:
out of standarization, should we also use here the `<action>HTTPHeaderInStringMap` functions? Are they any different than this implementation?

Reply:
It's because resource requests have `Headers map[string][]string`, whereas the others are `Headers map[string]string` 😭